src/linear_algebra/affine_space/independent.lean

```diff
@@ -239,6 +281,47 @@ by simp [ha]
 
 end affine_independent
 
+section field
+
+variables {k : Type*} {V : Type*} {P : Type*} [field k] [add_comm_group V] [module k V]
+variables [affine_space V P]
+include V
+
+/-- An affinely independent set of points can be extended to such a
+set that spans the whole space. -/
+lemma exists_subset_affine_independent_affine_span_eq_top {s : set P}
+    (h : affine_independent k (λ p, p : s → P)) :
```

Comment:
I don't think we use this extra indentation level, although I understand the intent.
```suggestion
  (h : affine_independent k (λ p, p : s → P)) :
```